### PR TITLE
feat: implement RegisterNodeService for UmeeApp

### PR DESCRIPTION
**Pull request by robert-zaremba**
## Description

Integrates new chain query service.
This is a new addon from SDK 0.46.3

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] added appropriate labels to the PR
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/umee-network/umee/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items._

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

**Comment by robert-zaremba** [comment]
seams we need to merge https://github.com/umee-network/umee/pull/1501 first

**Review by toteki — APPROVED** [review]
Seems very straightforward

File: app/app.go [hunk 1]
```diff
@@ -843,6 +847,10 @@ func (app *UmeeApp) RegisterTendermintService(clientCtx client.Context) {
 	)
 }
 
+func (app *UmeeApp) RegisterNodeService(clientCtx client.Context) {
+	nodeservice.RegisterNodeService(clientCtx, app.GRPCQueryRouter())
+}
```
Comment (toteki) [inline app/app.go:852]: did this function end up being required anywhere?

Comment (robert-zaremba) [reply app/app.go:852]: it's optional - when available,  SDK will register a query service to enable query chain info (like chain id)

**Pull request merged**